client/components/domains/contact-details-form-fields/style.scss

```diff
@@ -1,4 +1,4 @@
-.contact-details-form-fields {
+fieldset.contact-details-form-fields {
```

Comment:
This change introduced a bug in the first name field layout. We need to set the top margin for that field to be zero.